src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
I'm unsure about this, but this might violate the mod's coding standards. This should go right after the package if I'm not wrong. Try adding the mod's checkstyle file, it helps detect things like this!